weblate/screenshots/models.py

```diff
@@ -76,3 +83,24 @@ def change_screenshot_assignment(sender, instance, action, **kwargs):
         name="UnusedScreenshot"
     ).exists():
         component_alerts.delay([instance.pk])
+
+
+def update_screenshots_from_repo(self, component):
```

Comment:
This method will get called in `update_remote_branch` and then it will update the screenshots linked to the component?

Reply:
This should not have self arg, it's not a method. You don't need to call it directly there, but rather use signals to connect:

```suggestion
@receiver(component_post_update)
def update_screenshots_from_repo(sender, component, **kwargs):
```